scripts/submit-comment-with-artifact-links.mjs

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env node
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ */
+
+// @ts-check
+/* eslint-disable camelcase */
+
+// Submits a comment to the change PR or commit with links to artifacts that
+// show the results of the code change being applied.
+
+import dedent from "dedent";
+import { execSync } from "node:child_process";
+import { basename } from "node:path";
+import { Octokit } from "octokit";
+
+/**
+ * @type {Array<{path: string; url: string;}>}
+ */
+const { default: artifacts } = await import("./artifacts.json", { assert: { type: "json" }});
+
+if (artifacts.items === undefined) {
+    throw new Error(
+        "Unable to read artifacts.json, please make sure the CircleCI API call succeeded with the necessary personal access token.",
+    );
+}
+
+const ARTIFACTS = {
+    documentation: "packages/docs-app/dist/index.html",
+    landing: "packages/landing-app/dist/index.html",
+    table: "packages/table-dev-app/dist/index.html",
+    demo: "packages/demo-app/dist/index.html",
+};
+
+function getArtifactAnchorLink(pkg) {
+    const artifactInfo = artifacts.items.find(a => a.path === ARTIFACTS[pkg]);
+    return `<a href="${artifactInfo?.url}">${pkg}</a>`;
+}
+
+if (process.env.GITHUB_API_TOKEN) {
```

Comment:
looks like there are some old tokens in the CI env variables right now, I'm going to try those out. it those don't work though, I will have to generate & inject a new personal token just like https://github.com/palantir/blueprint/pull/5827 and the preview comments will come from my GitHub account

![image](https://github.com/palantir/documentalist/assets/723999/89ba4fb1-14a9-454a-b304-adc51e762542)

Reply:
yep, as I sorta expected, the old tokens didn't work. I'll switch to a new one

![image](https://github.com/palantir/documentalist/assets/723999/ad188fca-bc52-4ec5-aabf-f7f8f8f000da)